feature-group-definitions/import-maps.yml

```
@@ -1,4 +1,5 @@
 name: Import maps
+description: The `type="importmap"` attribute for the `<script>` HTML element indicates that the body of the element contains an import map. An import map is a JSON object that controls how the browser should resolve module specifiers when importing JavaScript modules.
```

I would avoid the word "body" which I associate with `<body>` or a fetch request/response body, while the children of an elements are its contents, or `textContent` property. How about:

```suggestion
description: A `<script type="importmap">` HTML element provides an import map as a JSON string. An import map controls how the browser should resolve module specifiers when importing JavaScript modules.
```